### DISCUSSION TITLE
Welcome to loro Discussions!

### DISCUSSION THREAD

**Discussion by zxch3n** [loro-dev/loro#253] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by ANUGLYPLUGIN** [comment]
Hello, How to join in this community by the way?

**Reply by zxch3n** [reply]
Hi! you can join our Discord channel https://discord.gg/tUsBSVfqzf

**Reply by ANUGLYPLUGIN** [reply]
Ok thanks

**Comment by SymphonyIceAttack** [comment]
I want to know how to find ecological libraries related to loro.js. This would be very frustrating if loro.js did not have a rich ecological library

**Reply by zxch3n** [reply]
We don't have one yet. You can use loro-crdt for now. Can you specify your frustration when using the package?
